docs/modules/reference/pages/provisioning/detectors/BsfDetector.adoc

```diff
@@ -15,43 +15,46 @@ The BSF Detector passes the `map`, `ip_addr`, `svc_name`, and `results` variable
 
 == Configuration and use
 
-.Optional parameters for the BSF service detector
+.Optional parameters for the BSFDetector
 [options="header"]
 [cols="1,3a,1"]
 |===
 | Parameter
 | Description
-| Default
+| Default value
 
 | fileName
-| The path to the file on the OpenNMS server which contains the BSF-compatible script to run.
+| Path to file on OpenNMS server containing BSF-compatible script to run.
 | none
 
 | langClass
-| The short name of the language the script is written in. Choices:
+| Short name of language the script is written in.
+Choices:
 
 * jython
 * beanshell
 * groovy
 | none
 
 | bsfEngine
-| The BSF engine to use when running the script. Choices:
+| BSF engine to use when running the script.
+Choices:
 
-* bsh.util.BeanShellBSFEngine
-* org.codehaus.groovy.bsf.GroovyEngine
-* org.apache.bsf.engines.jython.JythonEngine
+* `bsh.util.BeanShellBSFEngine`
+* `org.codehaus.groovy.bsf.GroovyEngine`
+* `org.apache.bsf.engines.jython.JythonEngine`
 | none
 
 | fileExtensions
-| A comma-separated list of file extensions to consider a valid script for this BSF instance.
+| Comma-separated list of file extensions be considered valid scripts for BSF instance.
```

Comment:
```suggestion
| Comma-separated list of file extensions to be considered valid scripts for BSF instance.
```
Neither this nor the original phrasing works completely. I'm trying to parse what we really need to say here:

1. Comma-separated list of file extensions.
2. These file extensions indicate the only file types that BSF instance recognizes as containing valid scripts. 

Maybe we need to make this into two sentences? Is there a way to rephrase to express those two ideas better?

Reply:
I think you're right about making it into two sentences. How's this look? :)

```suggestion
| Comma-separated list of file extensions.
By including them, you can indicate the only file types that the BSF instance recognizes as containing valid scripts.
```

Reply:
YES! Nice. Minor update below.

Reply:
> I think you're right about making it into two sentences. How's this look? :)

```suggestion
| Include them to indicate the only file types that the BSF instance recognizes as containing valid scripts.
```
```